Sources/StreamChat/Database/DatabaseSession.swift

```diff
@@ -167,6 +167,10 @@ protocol ChannelDatabaseSession {
         query: ChannelListQuery?
     ) throws -> ChannelDTO
     
+    /// Loads channel list query with the given filter hash from the database.
+    /// - Parameter filterHash: The filter hash.
+    func channelListQuery(filterHash: String) -> ChannelListQueryDTO?
+    
```

Comment:
Why was this added? 🤔 I don't see it being used

Reply:
Used in `ChannelListController.link/unlink` funcs 🙂